products/bash/install-ocp-pipeline.sh

```diff
@@ -24,7 +24,7 @@ metadata:
   name: openshift-pipelines-operator
   namespace: openshift-operators
 spec:
-  channel: stable
+  # channel: stable
```

Comment:
is there a reason for the channel to be commented out?

Reply:
With it commented out it used the default channel, which is usually the latest. I'm not seeing it on my current cluster, but on a previous cluster I think the stable channel didn't exist. Not sure if that was an OCP 4.12 thing. I think the change is safe, and we can re-test as part of the 2023.2.1 update, when we'll need to test on both 4.10 and 4.12

Reply:
Good point alright we'll leave it in then :)